cookbook-react/src/recipes/selectors/selector-2/index.tsx

```diff
@@ -0,0 +1,43 @@
+import React, { useState } from 'react';
+import cn from 'classnames';
+
+import './styles.scss';
+
+interface Option {
+  text: string;
+  disabled: boolean;
+  id: number;
+}
+
+interface Props {
+  options: Option[];
+}
+
+function Selector({
+  // TODO: Remove this options in a real app
+  options = [
+    { id: 1, text: 'Opción 1', disabled: false },
+    { id: 2, text: 'Opción 2', disabled: false },
+    { id: 3, text: 'Opción 3', disabled: false }
+  ]
+}: Props) {
+  const [active, setActive] = useState<null | number>(null);
```

Comment:
How do you use this component? How do you access the selected value?

Maybe make it a controlled component? 🤔

Reply:
Also add tests for this.